GraphQL-README.md

```diff
@@ -29,6 +29,7 @@ query BrandsQuery(
         commentary @include(if: $withCommentary) {
           rating
           embrace
+          embrace
```

Comment:
typo

Reply:
I tried creating an embrace campaign and querying a bank with it enabled. It returned "none" though, which doesn't seem right

Reply:
I tried add embrace campaign in existing brand and with creating new brand. It worked for me. Let me know if I am missing anything here.Below are the results of graphql query

query BrandByNameQuery($name: String!) {
  brandByName(name: $name) {
    tag
    commentary {
      embraceCampaign {
        id
      }
    }
  } 
}
  output : {
  "data": {
    "brandByName": {
      "tag": "hiver",
      "commentary": {
        "embraceCampaign": [
          {
            "id": "18"
          }
        ]
      }
    }
  }
}
------------------
{
  "data": {
    "brandByName": {
      "tag": "test_embrace_campaign",
      "commentary": {
        "embraceCampaign": [
          {
            "id": "18"
          }
        ]
      }
    }
  }
}